pkg/resolver/search.go

```diff
@@ -167,8 +177,13 @@ func (s *SearchResult) buildSearchQuery(ctx context.Context, count bool, uid boo
 
 	//WHERE CLAUSE
 	if s.input != nil && (len(s.input.Filters) > 0 || (s.input.Keywords != nil && len(s.input.Keywords) > 0)) {
-		whereDs = WhereClauseFilter(s.input)
-		sql, _, err := selectDs.Where(whereDs...).ToSQL()
+
+		//we get properties here as well as their datatypes:
+		whereDs, typeFilter = WhereClauseFilter(s.input, s.propTypes)
+		klog.V(7).Info("Property Datatype", typeFilter)
+
+	} else {
```

Comment:
What does the else clause mean here?

Reply:
This is why the RBAC clauses are excluded in the tests.

Reply:
Ah, okay. Not sure why it got excluded. Thanks for taking a look!